pkg/endpoint/endpoint.go

```diff
@@ -1360,7 +1363,8 @@ func FilterEPDir(dirFiles []os.FileInfo) []string {
 	eptsID := []string{}
```

Comment:
It's worth mentioning in the function comment that it now also returns `_next_failed` directories as well.